testdata/p4_16_samples/psa-unicast-or-drop-bmv2.stf

```diff
@@ -7,6 +7,9 @@ expect 2 000000000002 000000000000 ffff
 packet 4 000000000003 000000000000 ffff
 expect 3 000000000003 000000000000 ffff
 
-# This packet should be dropped
-packet 2 000000000000 000000000000 ffff
+# This packet should be dropped.  We send it into port 0, because if
+# there are no packets sent into nor expected on port 0, then the test
+# infrastructure does not check any of the packets that come out port
+# 0, or whether the right number come out port 0.
```

Comment:
Ahh we were unaware of this. So it would've simply passed since the check wouldn't have happened anyway. Is this specific to port 0; do you know if there are other ports that have this behavior?

Reply:
This is not specific to port 0.  If there is a port number mentioned in any packet or expect statement of an STF file, then the test infrastructure will already do complete checking of every packet that comes out of that port, or does not come out of that port but is expected to.

If a port number is mentioned in no packet or expect statement of an STF file at all, the test infrastructure will do absolutely no checking of any packets that come out of those ports, whether 0 packets come out, or 1000.